docs/source/installation/guide_quick.rst

```diff
@@ -40,24 +40,37 @@ If none of the lines show a red cross, indicating a problem, the installation wa
 
     If you encountered any issues, please refer to the :ref:`troubleshooting section <installation:troubleshooting>`.
 
-.. warning::
-
-    Not all AiiDA functionality is supported by the quick installation.
-    Please refer to the :ref:`section below <installation:guide-quick:limitations>` for more information.
-
 
 .. _installation:guide-quick:limitations:
 
 Quick install limitations
 =========================
 
+By default, ``verdi presto`` creates a profile that uses SQLite instead of PostgreSQL and does not use the RabbitMQ message broker.
+The table below gives a quick overview of the functionality that is not supported in those cases:
+
++-----------------------------------------+------------------------------------------------------------------------+
+| No RabbitMQ                             | SQLite instead of PostgreSQL                                           |
++=========================================+========================================================================+
+| Cannot run the daemon                   | Not suitable for high-throughput workloads                             |
++-----------------------------------------+------------------------------------------------------------------------+
+| Cannot submit processes to the daemon\* | No support for ``has_key`` and ``contains`` operators in query builder |
++-----------------------------------------+------------------------------------------------------------------------+
+| Cannot play, pause, kill processes      | No support for ``QueryBuilder.get_creation_statistics``                |
++-----------------------------------------+------------------------------------------------------------------------+
+
+\* Calculations can still be run on remote computers
+
+.. note::
+    To enable the RabbitMQ broker for an existing profile, :ref:`install RabbitMQ <installation:guide-complete:rabbitmq>` and then run ``verdi profile configure-rabbitmq``.
+
```

Comment:
```suggestion
By default, ``verdi presto`` creates a profile that can work without any server running in the background.

In particular:

- for the **database** (where the querable part of the data belonging to the AiiDA profile is stored): it uses SQLite instead of PostgreSQL.
- for the **broker** (that takes care of communication between processes): it does not use the RabbitMQ message broker.

For quick orientation, here is a matrix with the possible combinations of database and broker:

+----------------------+----------------------------------------------------+-------------------------------------------------------------+
|                      | **SQLite database**                                | **PostgreSQL database**                                     |
+======================+====================================================+=============================================================+
| **No broker**        | Quick start with AiiDA                             | [*not really relevant for any usecase*]                     |
+----------------------+----------------------------------------------------+-------------------------------------------------------------+
| **RabbitMQ broker**  | Production (low-throughput; beta, has limitations) | Production (maximum performance, ideal for high-throughput) |
+----------------------+----------------------------------------------------+-------------------------------------------------------------+

The table below gives a quick overview of the functionality that is not supported in those cases:

No broker (no RabbitMQ)  
-----------------------

No broker service needs to be installed and running on the system, however:

- Not suitable for high-throughput workloads (a poll-based mechanism is used rather than an event-based one)

- Cannot run the daemon (no `verdi daemon start/stop`), therefore processes cannot be submitted to the daemon (i.e., one can only use ``run()`` instead of ``submit()`` to execute calcuulations and workflows).
  **NOTE**: Calculations can still be run on remote computers.

- Cannot play, pause, kill processes.

.. note::
    Once you setup a profile with ``verdi presto``, you can at a later stage remove very simply the limitations of not having a broker. 
    This can be achieved by first installing the RabbitMQ broker (see :ref:`install RabbitMQ <installation:guide-complete:rabbitmq>`).
    Then, just run ``verdi profile configure-rabbitmq``.


SQLite instead of PostgreSQL (as the database)
----------------------------------------------

No database service needs to be installed and running on the system, however:

- Not suitable for high-throughput workloads (writes from multiple processes to the database are serialized)

- No support for ``has_key`` and ``contains`` operators in query builder 

- No support for ``QueryBuilder.get_creation_statistics``
```

Reply:
All the next points to be discussed:

* I tried to make the 2x2 matrix only to explain easily what each combination might be useful for. I convert the rest to a bullet-point list. 

* I also added for both that it's not suitable for high-throughput, for different reasons.

* I still added a 'beta' version to the RMQ+SQLite, as I think we are not 100% sure of all the missing functionalities I think. We can remove it after benchmarking. But open to discussion (I still have to read the rest of the docs, so feel free to change to make it homogeneous with the rest).